runhouse/servers/cluster_servlet.py

```diff
@@ -693,6 +730,10 @@ def status(self, send_to_den: bool = False):
     # Save cluster logs to Den
     ##############################################
     def _get_logs(self):
+        from pathlib import Path
+
+        if not Path(SERVER_LOGFILE).exists():
```

Comment:
is this expected to happen just initially? if the server doesn't exist at a certain point shouldn't we throw an exception

Reply:
yeah, the catchy thing is to determine if this is the first run of the logs loop or not. 
We could set a flag during init, and change it after the first run, even if the run failed (will change it in the finally block).